waiter/src/waiter/scheduler/kubernetes.clj

```diff
@@ -914,7 +922,7 @@
        :spec {:replicas min-instances
               :selector {:matchLabels {:app k8s-name
                                        :waiter/user run-as-user}}
-              :template {:metadata {:annotations {:waiter/port-count (str (+ ports offset))
+              :template {:metadata {:annotations {:waiter/port-count (str ports)
```

Comment:
Can we add in the top-level `[:metadata :labels]` a `:waiter/envoy-sidecar` entry similar to the `:waiter/fileserver` entry?

Then we can make assumptions about the ports being used based on the pod labels.

Reply:
Added new label `waiter/proxy-sidecar` with "enabled" / "disabled" value (just like the `waiter/fileserver` label).